Automattic/newspack-theme
# Add German translations to the theme

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds some baseline German translations to the theme.

Note: It should be tested with or after #743 so the json files are actually getting loaded. 

Closes #733.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Navigate to Settings > General, and change the language to German.
3. View the front-end of the site and confirm it's loading German strings -- some good spots to look are the copyright information in the footer, the edit link, or the byline on single posts.
4. Edit a single post; confirm that a couple of the Featured Image settings have been translated into German (not all of them had auto-generated translations).

### Other information:
